### PR TITLE
Polish the develop tutorial

### LINKED ISSUE #2077: Polish dev.md - how to build and contribute elasticdl.
How to build
- Pull elasticdl:dev from docker repo or build the image by our own.
- Build three pip packages using elasticdl:dev and build.sh

How to contribute
- Make code change locally
- How to run pre-commit locally
- How to run unit test locally
- Coding standard reference

### PR DESCRIPTION
**Pull request by workingloong**
Fix #2077

### CODE REVIEW

File: elasticdl/docker/Dockerfile [hunk 1]
```diff
@@ -40,7 +40,8 @@ RUN cd /tmp \
            https://github.com/tensorflow/tensorflow
 
 # Install Go and related tools
-ARG GO_MIRROR_URL=https://dl.google.com/go
+# ARG GO_MIRROR_URL=https://dl.google.com/go
```
Comment (brightcoder01) [inline elasticdl/docker/Dockerfile]: Please remove the commented code.

File: docs/tutorials/dev.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ cd elasticdl
 We prefer to install all building tools in a Docker image.
 
 ```bash
-docker build --target dev -t elasticdl:dev -f elasticdl/docker/Dockerfile .
+sh scripts/travis/build_images.sh
```
Comment (wangkuiyi) [inline docs/tutorials/dev.md]: How about we chmod +x build_images.sh, so we don't need to type `sh` here?

Comment (workingloong) [reply docs/tutorials/dev.md]: Good suggestion!

File: elasticdl/docker/Dockerfile (hunk 1 above)
Comment (workingloong) [reply elasticdl/docker/Dockerfile]: Done!

**Review by skydoorkai — APPROVED** [review]

**Pull request merged**